.github/workflows/codeql-analysis.yml

```diff
@@ -6,9 +6,11 @@ on:
       - develop
       - master
       - release/*
+      - topic/*
```

Comment:
What is this about?

Reply:
When I want to run CI on my on fork during development I need to either trigger it manually (workflow_dispatch), or push to a branch listed. I suggest adding `topic/*` as a prefix for feature (aka. topic) branches. GitHub workflow picks these settings up from the default branch making it tedious to maintain these settings locally.

Reply:
Can you please remove this? The project is using Git-Flow.

Reply:
Done.